Bio/Blast/NCBIXML.py

```diff
@@ -252,53 +252,53 @@ def _end_BlastOutput_version(self):
                 self._header.date = parts[2]
 
     def _end_BlastOutput_reference(self):
-        """A reference to the article describing the algorithm (PRIVATE).
+        """Yield the article describing the algorithm (PRIVATE).
 
         Save this to put on each blast record object
         """
         self._header.reference = self._value
 
     def _end_BlastOutput_db(self):
-        """The database(s) searched (PRIVATE).
+        """Yield the database(s) searched (PRIVATE).
 
         Save this to put on each blast record object
         """
         self._header.database = self._value
 
     def _end_BlastOutput_query_ID(self):
-        """The identifier of the query (PRIVATE).
+        """Yield the identifier of the query (PRIVATE).
 
         Important in old pre 2.2.14 BLAST, for recent versions
         <Iteration_query-ID> is enough
         """
         self._header.query_id = self._value
 
     def _end_BlastOutput_query_def(self):
-        """The definition line of the query (PRIVATE).
+        """Yield the definition line of the query (PRIVATE).
 
         Important in old pre 2.2.14 BLAST, for recent versions
         <Iteration_query-def> is enough
         """
         self._header.query = self._value
 
     def _end_BlastOutput_query_len(self):
-        """The length of the query (PRIVATE).
+        """Yield the length of the query (PRIVATE).
```

Comment:
Same as above, and applies to all the other private ``_end...`` methods which are used by the parse to capture/record information from the XML into our data structure.